windows/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM golang:1.18.4-nanoserver AS workspace
```

```suggestion
FROM golang:1.19-nanoserver AS workspace
```
was released few days ago

---

👍